### PR TITLE
Fidelia Nawar JavaScript HW

### PR DESCRIPTION
**Pull request by fidelianawar-zz**
Fidelia Nawar - 47963894

### CODE REVIEW

**Comment by fidelianawar** [comment]
Fidelia Nawar - 47963894

File: assignment.js [hunk 1]
```diff
@@ -176,7 +184,18 @@ assert.equal(nextExitLabel(),  "exit-1")
...
+  return function(parameter){
+    if(parameter != undefined){ //no param passed
+      count = parameter //if undef, assign count to param
+    }
+    else{ //param is not undef (param passed)
+      count++ //increment count by 1
+    }
+    res = label + "-" + count //format label to return
+    return res
+  }
```
Comment (pragdave) [inline assignment.js:198]: This is the first response I've seen that does it this way (keeping count to be the _previous number_, and not the next one). Cool.

File: assignment.js [hunk 2]
```diff
@@ -176,7 +184,18 @@ assert.equal(nextExitLabel(),  "exit-1")
...
+  let count = -1
+  return function(parameter){
+    if(parameter != undefined){ //no param passed
+      count = parameter //if undef, assign count to param
+    }
+    else{ //param is not undef (param passed)
+      count++ //increment count by 1
+    }
+    res = label + "-" + count //format label to return
+    return res
```
Comment (pragdave) [inline assignment.js:197]: Here, I'd just return the string directly: I don't see much point assigning it to a variable and then immediatelyu returning that variable.

Just an opinion: no deduction.

**Comment by pragdave** [comment]
# 100

**Pull request closed without merging**